### DISCUSSION TITLE
Questions regarding cron job on Mac?

### DISCUSSION THREAD

**Discussion by EdwinLapitan** [coccoinomane/crabada.py#91] [Q&A] [answered]
I have everything setup to the point where I can type in the commands to run all scripts. When following instructions for the bot to run without supervision, I get to the crontab and insert the lines as instructed but I get no feedback. I've read somewhere that once commands are in crontab that they are running but I'm not sure. Is there any way for me to see if it's actually running?

**Comment by loayei** [comment]
Yes you can open the terminal mail and you will see the messages there of the scripts that ran.
just type mail in the terminal and they should show up

**Reply by EdwinLapitan** [reply]
Thanks for getting back to me! Will update if all goes well!

**Reply by EdwinLapitan** [reply]
So the cron jobs are working. I've been banging my head against the wall trying to figure out why I keep getting this error.
Some help would be greatly appreciated!

Cron:
0,14 * * * * cd /Users/edwinlapitan/Desktop/crabada.py && /Library/Frameworks/Python.framework/Versions/3.10/bin/python3 -m bin.mining.sendTeamsMining <wallet address>
15,45 * * * * cd /Users/edwinlapitan/Desktop/crabada.py && /Library/Frameworks/Python.framework/Versions/3.10/bin/python3 -m bin.mining.closeMines <wallet address>

Error found in mail:
/Library/Frameworks/Python.framework/Versions/3.10/bin/python3: Error while finding module specification for 'bin.mining.sendTeamsMining' (ModuleNotFoundError: No module named 'bin')

**Comment by loayei** [comment]
'/Users/edwinlapitan/Desktop/crabada.py' try adding the apostrophe at the beginning and end. 
Not sure if that is the issue tho. Try it and let me know. 
You can also hit me up on discord ill try to help (loayei#1111)

**Reply by EdwinLapitan** [reply]
Thank you! I just sent you a friend request! Edwin#3117

**Comment by mertf** [comment]
I have my Python located in this folder on my Mac: /usr/local/bin/python3

You can perhaps try the same thing. You also need to give full disk access to terminal and/or cmd (I don't remember exactly the file names)

**Comment by coccoinomane** [comment, marked as answer]
Hi @EdwinLapitan !

Did you eventually solve your issue?

If not, you could try installing Python using this guide > https://docs.python-guide.org/starting/install3/osx/

Cheers,
Cocco

**Reply by EdwinLapitan** [reply]
Unfortunately, I have yet to figure out what I'm doing wrong. I resorted to the nuclear option and factory reset my mac, followed the Readme but still got the same error.  

/usr/local/opt/python@3.9/bin/python3.9: Error while finding module specification for 'bin.mining.sendTeamsMining' (ModuleNotFoundError: No module named 'bin')

**Reply by loayei** [reply]
I believe this should solve your issue.
Go to System Preferences on your mac then navigate to Security & Privacy then Scroll down to Full Disk Access and click the lock in the bottom to make changes. Make sure to tick cron then and close it.
Your issue should be solved now.

**Reply by EdwinLapitan** [reply]
HOLY SHIT IT WORKED!! THANKS A TON!!

I've been trying to get this to work for the past 3 days!! Much appreciated!!

**Reply by loayei** [reply]
Haha ! Glad that worked.
